data/korath/korath missions.txt

```diff
@@ -257,3 +257,53 @@ mission "Small Scale Delivery"
 	on complete
 		payment 1000
 		dialog `You find a Korath who seems to have been waiting for you in the spaceport. They greet you by showing their palms, then outstretch their hands as if waiting to receive the package. You hand off the small delivery and they hand you a small payment of <payment>.`
+
+mission "Korath Boarding Response"
+	invisible
+	boarding
+	repeat
+	minor
+	deadline 1
+	to offer
+		not "Korath Boarding Response: active"
+		or
+			and
+				random < 5
+				"reputation: Korath" < -10
+				"reputation: Korath" >= -100
+			and
+				random < 10
+				"reputation: Korath" < -100
+				"reputation: Korath" >= -1000
+			and
+				random < 25
+				"reputation: Korath" < -1000
+				"reputation: Korath" >= -10000
+			and
+				random < 50
+				"reputation: Korath" < -10000
+
+			and
+				random < 10
+				"reputation: Korath (Civilian)" < 0
+				"reputation: Korath (Civilian)" >= -300
+			and
+				random < 25
+				"reputation: Korath (Civilian)" < -300
+				"reputation: Korath (Civilian)" >= -3000
+			and
+				random < 50
+				"reputation: Korath (Civilian)" < -3000
+	source
+		government Korath "Korath (Civilian)"
+		system Alcyone Almach "Alpha Hydri" Atik Durax Misam Polaris Sheratan
+	on offer
+		conversation
+			"An angry hiss comes over your radio, and you see a strange pulse along the hull of the Korath vessel. Your ship detects a powerful hyperwave emission followed by several capital ships jumping in. The pulse throws your ship off of theirs, but it appears you could board them again. You had best finish your business here before you are reduced to a ball of superheated plasma."
```

Comment:
What's a hyperwave emission? Has it ever been mentioned in ES before?

Reply:
How should I know? I'm not a theoretical hyperparticle physicist.

Reply:
Just feels weird that this is mentioned in the game for the first time here.

Reply:
Ah, but that's only the beginning. I'm going to do a PR in a few days that'll mention steradians in a dramatic context.

Reply:
There are hyperwaves in #8075, also involving Korath.

Reply:
Ah ok then.